### PR TITLE
add meson package

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

File: meson.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package:
+  name: meson
+  version: 0.63.2
+  epoch: 0
+  description: Fast and user friendly build system
+  target-architecture:
+    - noarch
```
Comment (kaniini) [inline meson.yaml:7]: Wolfi doesn't have `noarch`.

File: meson.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - py3-setuptools
```
Comment (kaniini) [inline meson.yaml:22]: Shouldn't `python3` be here?

**Review by kaniini — CHANGES_REQUESTED** [review]

File: meson.yaml (hunk 2 above)
Comment (rawlingsj) [reply meson.yaml:22]: `python3` is coming in with `py3-setuptools`
```
apk add py3-setuptools
fetch https://packages.wolfi.dev/bootstrap/stage3/x86_64/APKINDEX.tar.gz
fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
(1/13) Installing libffi (3.4.2-r1)
(2/13) Installing gdbm (1.23-r1)
(3/13) Installing xz (5.2.6-r2)
(4/13) Installing mpdecimal (2.5.1-r1)
(5/13) Installing readline (8.1.2-r1)
(6/13) Installing sqlite (3.39.2-r1)
(7/13) Installing python3 (3.10.7-r0)
(8/13) Installing py3-appdirs (1.4.4-r0)
(9/13) Installing py3-more-itertools (8.13.0-r0)
(10/13) Installing py3-ordered-set (4.0.2-r0)
(11/13) Installing py3-parsing (3.0.9-r0)
(12/13) Installing py3-packaging (21.3-r0)
(13/13) Installing py3-setuptools (59.4.0-r0)
Executing glibc-2.36-r0.trigger
Executing busybox-1.35.0-r2.trigger
OK: 2509 MiB in 53 packages
```

I wasn't sure what the right thing to do was, shall I be explicit and add `python3` here?

File: meson.yaml (hunk 1 above)
Comment (rawlingsj) [reply meson.yaml:7]: Ok great - should I change this to `all`?

Comment (kaniini) [reply meson.yaml:7]: Yes!

File: meson.yaml (hunk 2 above)
Comment (kaniini) [reply meson.yaml:22]: Yes, I would be explicit and add python3 there, just to make it clear that this is a python-based tool.

Comment (rawlingsj) [reply meson.yaml:22]: perfect - that makes sense

File: meson.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package:
+  name: meson
+  version: 0.63.2
```
Comment (eli-schwartz) [inline meson.yaml:3]: Since you wrote this, we've released version 0.63.3

File: Makefile [hunk 4]
```diff
@@ -135,5 +135,6 @@ $(eval $(call build-package,grype,0.50.2-r0))
 $(eval $(call build-package,trivy,0.32.0-r0))
 $(eval $(call build-package,ruby,3.1.2-r0))
 $(eval $(call build-package,rust-stage0,1.64.0-r0))
+$(eval $(call build-package,meson,0.63.2-r0))
```
Comment (kaniini) [inline Makefile:138]:
```suggestion
$(eval $(call build-package,meson,0.63.3-r0))
```

**Review by kaniini — CHANGES_REQUESTED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**